_assets/scripts/run_unit_tests.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+
+UNIT_TEST_PACKAGES=$(go list ./...  | \
+  grep -v /vendor | \
+  grep -v /t/e2e | \
+  grep -v /t/benchmarks | \
+  grep -v /transactions/fake )
+
+for package in ${UNIT_TEST_PACKAGES}; do
+  echo "Testing ${package}"
+  package_dir=$(go list -f "{{.Dir}}" "${package}")
+  output_file=${package_dir}/test.log
+
+  go test -tags "${BUILD_TAGS}" -timeout 30m -v -failfast "${package}" ${GOTEST_EXTRAFLAGS} | \
+     if [ "${CI}" = "true" ]; then cat > "${output_file}"; else tee "${output_file}"; fi
+  go_test_exit=$?
+
+  if [ "${CI}" = "true" ]; then
+    go-junit-report -in "${output_file}" -out "${package_dir}"/report.xml
+  fi
+
+  if [ ${go_test_exit} -ne 0 ]; then
+    echo "  Failed, see the log: ${output_file}"
```

Comment:
This is good, but it would be nice to make it a bit more visible for people:
```suggestion
    echo -e "${YLW}Failed, see the log:${RST} ${BLD}${output_file}${RST}"
```
Of course we'd need the colors:
```sh
export YLW='\033[1;33m'
export RED='\033[0;31m'
export GRN='\033[0;32m'
export BLU='\033[0;34m'
export BLD='\033[1m'
export RST='\033[0m'
```

Reply:
The colors could be in a separate file, as we do it in `status-mobile`:
https://github.com/status-im/status-mobile/blob/develop/scripts/colors.sh

Reply:
Good idea, let me try, how it will work on CI and locally.